### ISSUE TITLE
Clarification of the normative status of the termlist document

### ISSUE THREAD

**Issue by baskaufs** [tdwg/ac#109]
In order to make the [term list document](https://github.com/tdwg/ac/blob/documentation-conversion/doc/termlist.md) conform to the Standards Documentation Specification, it needs a statement in its introduction indicating what parts of it are normative.  

The abstract of the document says "This is a normative document and the definition may not be changed without due process."  So clearly some parts of the document should be considered normative.  It is possible that only the ac: namespace term definitions themselves are normative in this document (that's suggested by the wording of the abstract).  However, it is more likely that other parts of the document should also be considered normative.  

[Section 2 of the Structure document](https://github.com/tdwg/ac/blob/documentation-conversion/doc/structure.md#2-terminology-of-this-specification) states clearly that term labels are only suggestions, so they should NOT be considered normative.

Section 1.5 of the [Standards Documentation Specification](https://github.com/tdwg/vocab/blob/master/sds/documentation-specification.md#15-definitions) defines "normative" as "prescriptive parts of a standard that specify features, characteristics, or behaviors that are necessary to comply with the standard".  So any part of this documentation that tell users how to comply with the standard should be considered normative.  This gets a bit complicated with the borrowed terms because the definitions given in this document explain how to use borrowed terms in a way that complies with the AC standard, but they are not synonymous with the definitions given by the vocabularies that actually define those terms.  So are the AC definitions normative, given that AC is not the authoritative organization that has minted those terms?

One thing that needs to be considered in this document is that there will be parallel machine-readable metadata descriptions for each term.  So anything that shows up in the term tables will have a machine-readable analog.  [Section 4.5 of the SDS](https://github.com/tdwg/vocab/blob/master/sds/documentation-specification.md#45-metadata-properties-for-describing-vocabulary-terms) RECOMMENDS that the rdfs:comment property be used to indicate the term definition.  It also says that "the property dcterms:description MAY optionally be used to provide additional information that is not part of the definition of the term."  It is silent about the use of other descriptive properties.  It may be a good idea to use one or more of the [SKOS Documentary Notes](https://www.w3.org/TR/2009/NOTE-skos-primer-20090818/#secdocumentation) properties to provide some of this additional information.  In particular, skos:scopeNote and skos:example might be useful.  A combination of use of these terms with dcterms:description might be sufficient to present the normative and non-normative information about non-AC-defined terms, allowing us to stick to the definition given by the defining authority as the value for rdfs:comment.  One nice thing about those particular SKOS properties is that they are do NOT have any explicit range or domain properties.  So they can be used as properties of any resource without entailing any "naughty" things that we don't intend.  For example, using them with a term does NOT imply that the term is a SKOS concept.

The other problem that was mentioned at the 2018-07-18 hangout was that the current wiki-based documentation depends heavily on footnotes included in the term metadata sections.  This information will be lost when the machine-readable version is created, and the SDS requires that the machine-readable representation contain substantively identical information to the human-readable representation.  So I think that we just need to make sure that whatever important information is included in the footnotes just gets directly inserted into one of the term metadata values.

I'm not going to try to impose any kind of timeline on this issue because I think it is critical to get it right.  If we can't come to some kind of consensus before the Dunedin meeting, I think we should try to have some kind of informal hack-a-thon there where anyone who cares can hammer out a fix to this in person.

**Comment by baskaufs** [comment]
This issue is related to #90, which should be resolved at the same time as this one.

**Comment by baskaufs** [comment]
The endnotes included in the termlist.md footer have now been rendered unnecessary by including their information at the place where they were referenced.  

There is now a [description of what parts of the terms tables are normative](https://github.com/tdwg/ac/blob/documentation-conversion/doc/termlist.md#11-status-of-the-content-of-this-document). That section currently says "the values of the Normative URI, Definition, Layer, Required, and Repeatable are normative. The values of Term Name is non-normative, although one can expect that the namespace abbreviation prefix is one commonly used for the term namespace. Labels and the values of all other properties (such as notes) are non-normative."  The "all other properties (such as notes)" gives us the option of breaking down the "Notes" section into more fine-grained categories, such as skos:example or rdfs:seeAlso (?) if we wish.  

We still need to deal with the problem that the definitions we give for non-AC-defined terms are not the definitions given in the vocabulary that minted them (which should be normative for them).  I feel strongly that if we say that the "Definition" value is declared to be normative, then we need to provide the value given in the source vocabulary.  My current thinking is that what we currently have under definition is actually a [skos:scopeNote](https://www.w3.org/TR/skos-primer/#secdocumentation).  If that is true, do we want to say that our scope note is also normative (i.e. in that field we say how the term should be used to conform with AC's adoptive use)?

Please make comments in the tracker for this issue if you have an opinion about this.

**Comment by nielsklazenga** [comment]
I like the idea to use the definitions from the standards we borrow the terms from and separate the current definition as a scopeNote. I think, at the moment, non-normative, like the other notes. We can always make individual scopeNotes normative if they really restrict the scope of the original concept.

**Comment by baskaufs** [comment]
I am thinking that the scopeNote plan is the simplest way to separate out the authoritative definitions given by the term creators from the AC-specific notes.  Probably going beyond that (to separating out examples, etc. is beyond the scope of bringing the document into compliance with the SDS.  When I get some time, I'll create a new column in the rs.tdwg.org repo tables for the authoritative definitions, look them all up, and paste them into the table.  After that, I'll fix the page-generating script to display both the authoritative definition and the AC scopeNotes.

**Comment by baskaufs** [comment]
OK, so I've finished adding to the rs.tdwg.org repo the authoritative borrowed term definitions taken from the documentation that defined them.  The script that generates the termlist.md document now adds an row called "Usage:" to some terms.  If the term definition already contained the authoritative definition (as was the case for all of the ac: terms and most or all of the dwc: terms), then there is no Usage: row - the previous definition stands.  For non-TDWG borrowed terms, the Definition: row now contains the normative definition taken from the documentation provided by the organization that minted the term.  What was previously in the Definition: row for those terms has now been moved to the new Usage: row.  

So there are three remaining components of this that still need to be resolved:

1. The borrowed dwc: terms need to be checked to see if their Definition entry matches their definition in Darwin Core.  If not, then the AC definition for that term needs to be moved to Usage and the authoritative definition needs to be put in the definition.  The Geography Vocabulary terms that were not explicitly listed in the original documentation have the authoritative definitions, but the ones that were listed in the original term list document need to be checked.
2. We need to decide on the normative/non-normative status of the new Usage: rows.  I think they should be normative because they were considered the term "definition" in the previous termlist document.  Essentially, they prescribe how those borrowed terms should be used in the context of AC.  So unless I hear objections in the near future, I'll specify that Usage: is normative in section 1.1 of the introduction.
3. We need to check on the status of the missing hyperlinks in the Notes section.  There were many of them in the original document that didn't survive the transition to the plain text CSV files that are being used to generate the Notes in the new document.  The solution would be to just put the full URLs in parentheses within the notes.  Then they would persist in the machine-readable representations and in the human-readable documentation they would show up as hyperlined when the Markdown was rendered.

I think when these three things are done, we should be able to merge the documentation-conversion branch into the master branch.

**Comment by baskaufs** [comment]
@ramorrismorris In the [notes on ac:subtype](https://terms.tdwg.org/wiki/Audubon_Core_Term_List#ac:subtype), it says "Conforming applications may choose to ignore these."  What are "these"?  Any assigned subtype values (whether literals or URIs)?   URI values of ac:subtype only (this statement is not made in the notes for ac:subtypeLiteral)?

**Comment by baskaufs** [comment]
OK, I feel that I have addressed these issues satisfactorily as far as conversion to SDS conformance is concerned.  In some cases where examples were clearly misplaced in definitions, I moved them to the Notes section.  However, there are many places in the documentation where it is not clear whether certain content in the Definition, Usage, or Notes fields are misplaced.  I think we need to have a follow-up effort to go through the entire AC vocabulary to look at this.  At the same time, we should examine the issue of whether we are using borrowed terms faithfully with their definitions (the essence of issue #88).  

I am going to close this issue now, as well as #90, which has been resolved (well enough for now).
